### ISSUE TITLE
Find & add additional modes

### ISSUE THREAD

**Issue by AdamNiederer** [AdamNiederer/vue-mode#57]
I'm not very familiar with exactly what templating and scripting and styling languages are supported by the vue template compiler, so I could use some help ensuring our list of available major modes is up to date. 

That list can be found here: https://github.com/AdamNiederer/vue-mode/blob/master/vue-mode.el#L42

The format is pretty simple: For the opening tag ` <:type lang=":name">`, vue-mode finds an entry which matches both `:type` and `:name`, and applies the mode specified by `:mode` until the closing tag.

In short, find what other languages are supported by the vue template compiler, find a corresponding major mode for emacs, and add it to the list. Alternatively, find some aliases for already-supported languages and add those to the list along with the existing entries.
